FactoryMethod/src/main/java/com/ben/abstractnotification/AbstractFactoryPatternUML.puml

```diff
@@ -0,0 +1,56 @@
+@startuml
+header AbstractFactoryPattern
```

Comment:
I have to say I am really loving these UML diagrams @BenRodriguez10 ! I saw another really interesting tool called [Structurizr](https://structurizr.com) that allows you to write architecture diagrams as code too

Reply:
Ayy will take a look!